File: src/main/java/ru/mydesignstudio/database/metadata/extractor/extractors/trigger/TriggerModel.java
package ru.mydesignstudio.database.metadata.extractor.extractors.trigger;

import lombok.Data;

@Data
public class TriggerModel {
  private String triggerSchemaName;
  private String triggerName;
  private String triggerType;
  private String triggeringEvent;
  private String schemaName;
  private String objectName;
  private String objectType;
  private String status;
  private String script;
}

File: extractor-core/src/main/kotlin/ru/mydesignstudio/database/metadata/extractor/extract/result/TriggerModel.kt
package ru.mydesignstudio.database.metadata.extractor.extract.result

data class TriggerModel(
        val triggerSchemaName: String,
        val triggerName: String,
        val triggerType: String,
        val triggeringEvent: String,
        val schemaName: String,
        val objectName: String,
        val objectType: String,
        val status: String,
        val script: String)